### PR TITLE
[14.0][MIG] connector_sage

### PR DESCRIPTION
**Pull request by eantones**
Supersedes #87

### CODE REVIEW

File: connector_sage/models/hr_employee/binding.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Copyright NuoBiT Solutions - Eric Antones <eantones@nuobit.com>
+# Copyright NuoBiT Solutions - Kilian Niubo <kniubo@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class HrEmployee(models.Model):
```
Comment (eantones) [inline connector_sage/models/hr_employee/binding.py:8]: @KNVx this class has no "_description" field and there's other PRs with the same problem. Please run each of your PR's in a test environment and check the log warnings in order to know what to fix. Thanks

**Comment by LoisRForgeFlow** [comment]
@MiquelRForgeFlow can you add the needed migration script to force the update of the multicompany rules of this module: https://github.com/nuobit/odoo-addons/pull/93/files#diff-6b160955f2bf9236174daa71d0ea8057d49ed8528f9adeed0495193eea33ffa4R11?

Thanks

**Comment by MiquelRForgeFlow** [comment]
@LoisRForgeFlow Done in https://github.com/nuobit/odoo-addons/pull/137.

**Comment by MiquelRForgeFlow** [comment]
Selection_342

**Pull request merged**